platform/ios/CHANGELOG.md

```diff
@@ -4,10 +4,14 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 
 ## 4.0.1
 
+### Packaging
+
+* Re-added support for 32-bit simulators (i386) to avoid conflicts with CocoaPods. ([#11891](https://github.com/mapbox/mapbox-gl-native/pull/11891))
```

Comment:
Maybe word it more strongly, like “to work around an issue in CocoaPods”, since it doesn’t seem to be a bug on our end?